resources/commitee-documentation/membership-committee-charter.md

```diff
@@ -0,0 +1,96 @@
+<p align="center">
+  <img src="../../img/dotnetfdn.png" />
+</p>
+
+# Membership Commitee Charter
+
+
+## Mission Statement
+
+To manage and improve the membership offering of the .NET Foundation.
+
+## Membership
+
+The option to take part and be involved in the membership committee is open to all members of the .NET Foundation. 
+
+We ask that you have a GitHub profile set up, as our organisation and tooling is all based in this GitHub repository. 
+
+### How To Join The Membership Committee
+
+- [Create an issue](https://github.com/dotnet-foundation/wg-membership/issues/new/choose) using the "Membership" issue template
+
+<p align="center">
+  <img src="../../img/membership-request.png" />
+</p>
+
+- On seeing this [Jessica White](https://github.com/wordshaker) will accept the issue and close it, then send you the next meetings invite.
+- After attending two meetings, you will be added to the CODEOWNERS of this repository and be added to the committee GitHub Teams.
+
+### Expected Level of Contribution
+
+To remain a member of the committee we ask that you attempt to attend meetings at least once a month. If you are unable to attend at this frequency or have to miss a couple for extenuating circumstances, you will be expected to raise this with the committee. You won't need to provide specifics, it's to let the group know you intend to take part in future.
+
+If you cannot attend at that frequency, you could also contribute by submitting issues or pull requests to the repository.
+
+Each committee meeting will have a draft agenda in GitHub available before the meeting. This is how to check when issues you are passionate about are going to be up for discussion and action votes.
+
+## Termination of Membership
+
+On a quarterly basis, the chair, scribe or appointed representative will review levels of contribution from [members of the membership GitHub Teams](https://github.com/orgs/dotnet-foundation/teams/membership/members). If the expected contribution levels have not been met, with no reasoning, your membership to this team will be revoked.

```

Comment:
I like this. It's important that the list of committee members accurately reflect who is actively participating. I liked your idea on another thread to provide some notice when this happens.

Reply:
https://github.com/dotnet-foundation/wg-membership/pull/89/commits/e738709f83df3f88f31328b0350e7731ceb549c7